### PR TITLE
Data DAO concepts - initial page

### PR DESCRIPTION
**Pull request by ElPaisano**
[Checkpoint view requested.](https://github.com/filecoin-project/filecoin-docs/pull/1926#issuecomment-1507669072)

Addresses https://github.com/filecoin-project/filecoin-docs/issues/1913. 

Scope: A simple, relatively short, and focused page on the what / why / how do I get started with Data DAOs from a high level. 

Not ready for final review.

### CODE REVIEW

**Comment by ElPaisano** [comment]
Input requested:

What other information should be included here?

Also, is the audience for this page only developers seeking to build data DAOs? Or are we gearing this more towards anyone familiar with Filecoin / who uses Filecoin?

- It would be good to include a section discussing value / problems solved by data daos.

- I think the Prerequisites info should be included. Can refactor as tooltips or @johnnymatthews suggested [<details> dropdown](https://filecoinproject.slack.com/archives/C03Q117UBLM/p1681338078576009?thread_ts=1681263782.663959&cid=C03Q117UBLM). Thoughts?

- I'd like to expand on the existing example. Ideally, make it seem realistic and relatable, with examples of each of the design principles listed in **Designing a Data DAO**. While my example is _ok_, I think we could do better here.

- I know that tutorials / guides / demos and all that are still being worked out, but it would be good to wrap this section up with a **Next Steps** section (or something similar) that outlines the different solutions for data dao developers

File: content/en/basics/what-is-filecoin/data-daos/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: "Data DAOs"
+description: "A blockchain is a distributed database that is shared among the nodes of a computer network. This page covers how the Filecoin blockchain is designed, and the various functions it has."
```
Comment (ElPaisano) [inline content/en/basics/what-is-filecoin/data-daos/index.md:3]:
```suggestion
description: "Learn Data DAO key concepts and how to get started with Data DAOs."
```

**Comment by elizabeth-griffiths** [comment]
@ElPaisano I don't think we've met yet. I'm the NFT.Storage Product Manager. I've been reaching out to teams who are building, or plan to build data daos with the goal to learn from them and potentially collaborate with them.
Regarding your question about if  https://filecoin.io/blog/posts/fvm-imaginarium-bringing-datadaos-to-the-next-level-with-fvm/ is up to date, I think it needs as least one update, possibly more. It seems like the Kangaroo team may not be actively building anymore. Any chance you're in touch with them?

**Comment by johnnymatthews** [comment]
We can use the existing `{{< details "Title" >}}` shortcode for this:

Here's an example:

````markdown
{{< details "This is a detail block" >}}
Here is some content for the inside of the details block.

```plaintext
It looks like code renders normally.
```

1. And so
1. Do
1. Lists

---

![This is an image.](https://placehold.co/200x100)

> This is a quote block.
{{< /details >}}
````

Which spits out something like this:

Screenshot 2023-04-14 at 16 33 47

**Comment by ElPaisano** [comment]
> We can use the existing `{{< details "Title" >}}` shortcode for this:

@johnnymatthews nice this looks good

File: content/en/basics/what-is-filecoin/data-daos/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  basics:
+    parent: "basics-what-is-filecoin"
+    identifier: "data-daos-0b1a5bc9c50789118de7a7fbba355514"
+weight: 90
+toc: true
+aliases:
+    - "/intro/intro-to-filecoin/data-daos/"
+---
+
+<details>
```
Comment (johnnymatthews) [inline content/en/basics/what-is-filecoin/data-daos/index.md:18]:
```suggestion
<details>
{{< details "Data DAOs" >}}
```

File: content/en/basics/what-is-filecoin/data-daos/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+Before reading this page, ensure that you are familiar with the following concepts:
+
+- **Decentralized autonomous organizations (DAOs)**: Distributed entities whose ownership, governance, and decision-making processes are transparently conducted via on-chain activities, and are tasked with a collective mission.
+
+- **Smart contracts**: Self-executing programs that operate on the Filecoin Virtual Machine and are used to automate the enforcement and execution of agreements between network participants. Smart contracts on the Filecoin network can be used to manage storage and retrieval deals, allocate storage resources, and facilitate the governance of decentralized autonomous organizations (DAOs) and other decentralized applications (dApps).
+</details>
```
Comment (johnnymatthews) [inline content/en/basics/what-is-filecoin/data-daos/index.md:24]:
```suggestion
{{< /details >}}
```

File: content/en/basics/what-is-filecoin/data-daos/index.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    parent: "basics-what-is-filecoin"
+    identifier: "data-daos-0b1a5bc9c50789118de7a7fbba355514"
+weight: 90
+toc: true
+aliases:
+    - "/intro/intro-to-filecoin/data-daos/"
+---
+
+{{< details "Data DAOs" >}}
+Before reading this page, ensure that you are familiar with the following concepts:
```
Comment (johnnymatthews) [inline content/en/basics/what-is-filecoin/data-daos/index.md:19]:
```suggestion
{{< details "Before reading this page, ensure that you are familiar with the following concepts:" >}}
```

**Comment by johnnymatthews** [comment]
This can probably merge now, right @ElPaisano ?

**Comment by ElPaisano** [comment]
@johnnymatthews this is still missing some input and info from the team. I also am not sure what info should be in the Next Steps section https://github.com/filecoin-project/filecoin-docs/pull/1926/files

@hammertoe @trruckerfling do y'all have anything else you'd like to see with this?

**Comment by johnnymatthews** [comment]
Converting to draft.

**Pull request closed without merging**